vocab.md

```diff
@@ -18,10 +19,14 @@ This is a list of new technical vocabulary we come across during the React unit.
 - Bundler
     - Webpack
     - Source Maps
+- ES6
 
 ## React Vocab
 - JSX
 - Virtual DOM
 - React Component
 - React Element
 - React Root
+- classname
+- Relative Path
+- Absolute Path
```

Comment:
Could you move these terms to the software and CS section?

Reply:
Done and added a couple of more words from the props lesson